tests/cloud_playbooks/roles/packet-ci/tasks/create-vms.yml

```diff
@@ -30,15 +30,19 @@
     index_var: vm_id
 
 - name: Wait for vms to have ipaddress assigned
-  shell: "kubectl get vmis -n {{ test_name }} instance-{{ vm_id }} -o json | jq '.status.interfaces[].ipAddress' | tr -d '\"'"
```

Comment:
A `| cut -d/ -f 1` might do the trick here?

Reply:
Probably, but I'm not certain that the field we capture is really the VM's IP.